### PR TITLE
[Feature][AES-2550] Update the style of checkbox and error message

### PR DESCRIPTION
**Pull request by zack76**
Update the style of checkbox as well as the error messages
image
image

### CODE REVIEW

File: src/components/Checkbox/Checkbox.module.css [hunk 1]
```diff
@@ -56,22 +56,29 @@
   &:hover {
     cursor: pointer;
   }
-
-  &.hasError {
-    border-color: var(--color-error);
```
Comment (MkMan) [inline src/components/Checkbox/Checkbox.module.css:61]: Was this requested by the UX team?

Comment (zack76) [reply src/components/Checkbox/Checkbox.module.css:61]: > Was this requested by the UX team?

Yes, the checkbox doesn't need to be in red color in this case

**Review by MkMan — APPROVED** [review]

File: src/components/Checkbox/Checkbox.module.css [hunk 2]
```diff
@@ -56,22 +56,25 @@
...
 
 .content {
   margin-left: 10px;
   font-size: rem(14px);
   line-height: 2;
+
   &.dark {
     color: var(--color-dark-copy);
+    a {
+      color: var(--color-dark-copy);
```
Comment (kevin-ho87) [inline src/components/Checkbox/Checkbox.module.css:69]: you can do `color: currentColor` so it inherits the parent colour. Simpler i feel

Comment (zack76) [reply src/components/Checkbox/Checkbox.module.css]: > you can do `color: currentColor` so it inherits the parent colour. Simpler i feel

Thank you! Updated.

Comment (kevin-ho87) [reply src/components/Checkbox/Checkbox.module.css]: Just make sure it still works with the 'light/dark' theme haha

**Review by kevin-ho87 — APPROVED** [review]

File: src/components/Checkbox/Checkbox.module.css (hunk 2 above)
Comment (zack76) [reply src/components/Checkbox/Checkbox.module.css]: > Just make sure it still works with the 'light/dark' theme haha

yeah, i tested it and all look good

**Pull request merged**